force_wfmanager/wfmanager_review_task.py

```diff
@@ -45,9 +44,6 @@ class WfManagerReviewTask(Task):
     #: and table
     analysis_model = Instance(AnalysisModel, allow_none=False)
 
-    #: Is the 'run' toolbar button active
-    run_enabled = Bool(True)
-
     #: Are the saving and loading menu/toolbar buttons active
     save_load_enabled = Bool(True)
```

Comment:
I think this attribute `save_load_enabled` can now be removed, along with the `sync_save_load_enabled` method on line 305